cmd/create/network/cmd_test.go

```diff
@@ -0,0 +1,64 @@
+package network
+
+import (
+	"go.uber.org/mock/gomock"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/pkg/network"
+	bsOpts "github.com/openshift/rosa/pkg/options/network"
+)
+
+var _ = Describe("Network", func() {
+	var ctrl *gomock.Controller
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+	})
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	Context("Network Stack", func() {
+		It("should create a stack successfully", func() {
+			serviceMock := network.NewMockNetworkService(ctrl)
+			serviceMock.EXPECT().CreateStack(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).Times(1)
+			err := serviceMock.CreateStack("example.yaml", nil, nil)
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
+})
+
+var _ = Describe("Validation functions", func() {
+	var (
+		ctrl     *gomock.Controller
+		mockArgs *bsOpts.NetworkUserOptions
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockArgs = &bsOpts.NetworkUserOptions{}
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	Context("parseParams", func() {
+		It("should correctly parse the tags and parameters", func() {
```

Comment:
i think a negative test would be good here, related to above comment (if need be)